### DISCUSSION TITLE
Vercel Templates

### DISCUSSION THREAD

**Discussion by benjvelt** [vercel/community#3969] [Feature Question]
### Question

Hey Vercel Team,

How can I create a Vercel Template such as the ones listed here: [](https://vercel.com/templates)?

Is there a process for submission?

Thanks,
Ben

**Comment by amyegan** [comment]
There isn't currently any formal process for template submission. The [Template](https://vercel.com/templates) projects are typically created by Vercel staff and partners to demonstrate specific features.

However, you can add a [Deploy Button](https://vercel.com/docs/deployments/deploy-button#) to your repo to make it easy for others to use as a starting point for their own projects. 😃

**Reply by Mikalis** [reply]
@amyegan any update on that? @leerob mentioned we, the community, can also soon upload templates?

**Reply by amyegan** [reply]
Thanks for your interest in adding a template to our Template Marketplace! We are not currently accepting new templates, but we are reimagining what a new Template Marketplace could look like. If that change happens, you will see an announcement from our social media accounts.

**Comment by Mikalis** [comment]
awesome!

Amy ***@***.***> schrieb am Fr., 20. Okt. 2023, 17:49:

**Comment by amyegan** [comment]
We just re-opened Templates Marketplace for new submissions. If you're still interested in sharing your template, please use the [Templates Marketplace submission form](https://vercel.com/templates/submit) to send us the details. 🙂
